Jenkinsfile

```
@@ -556,7 +169,9 @@ pipeline {
                 stage("HIP Package") {
                     agent{ label rocmnode("nogpu") }
                     steps{
-                        buildHipClangJobAndReboot( package_build: "true", needs_gpu:false, needs_reboot:false)
+                        script {
+                        utils.buildHipClangJobAndReboot( package_build: "true", needs_gpu:false, needs_reboot:false, needs_cleanup:true)
```

```suggestion
                        utils.buildHipClangJobAndReboot( package_build: "true", needs_gpu:false, needs_reboot:false)
```
the "needs_cleanup" arg can be removed everywhere. It is automatically enabled and RunPerf doesnt need it